software/verification/verify_cdl_sequence.py

```diff
@@ -94,23 +96,53 @@ def parse_configurations(self):
                 else:
                     reference['modelJsonDirectory'] = default_model_json_directory
 
-            if not reference['modelJsonDirectory'].endswith('/'):
-                reference['modelJsonDirectory'] += '/'
+            if not reference['modelJsonDirectory'].endswith(os.sep):
+                reference['modelJsonDirectory'] += os.sep
 
             if ref_run_controller_flag:
                 reference['controller'] = self.real_controller
 
             self.test_list.append(reference)
 
+    def run_modelica_json(self, model, model_dir):
+        """this function uses the modelica-json tool to convert modelica files to json
+
+        Parameters:
+        -----------
+        model : str
+                name of modelica model to be translated to json
+        model_dir : str
+                    directory where to store the json output
+        """
+        app_js_path = os.path.join(self.modelica_json_path, 'app.js')
+        model_path = model.replace('.', os.sep) + '.mo'
+        print("running modelica-json tool for {0}".format(model))
+        result = subprocess.run(['node {0} -f {1} -d {2} -o json'.format(app_js_path, model_path, model_dir)],
+                                stdout=subprocess.PIPE, shell=True)
```

Comment:
Do you really need `shell=True`? I think this can be done without it, see https://docs.python.org/3/library/subprocess.html#security-considerations